draft-lenders-dns-over-coap.md

```diff
@@ -353,6 +353,40 @@ Reference: [TBD-this-spec]
 
 --- back
 
+Other CoAP Request Methods
+==========================
+This document only introduces the usage of the FETCH method for CoAP requests
+carrying DNS queries. If for whatever reason other methods are required to be
+used, request methods that can carry payloads, such as POST or PUT, are
```

Comment:
I'm fine with this appendix, but why is PUT introduced? Unlike POST which has intentionally vague semantics, PUT has defined semantics ("that the resource [...] be updated or created with the enclosed representation") that don't match what's being done here.

Reply:
i agree that this approach could be a good compromise. but this also means that we abandon the C-H-Proxy argument right away.

Reply:
> I'm fine with this appendix, but why is PUT introduced?

I guess I was in "people might do anything if they don't get propper instructions"-mode here. I can remove it again, if it is more distracting than useful.

> but this also means that we abandon the C-H-Proxy argument right away.

I was thinking about adding a sentence or two about how C-H-Proxies could be implemented in §5. Of course, for this recommendations should be amended here as well then.

Reply:
we cannot propose  C-H-Proxies in the body of the draft and specify the required building blocks in the Appendix. if we move GET to the Appendix, the proxy option is out-of-scope for the draft.

Reply:
> we cannot propose C-H-Proxies in the body of the draft and specify the required building blocks in the Appendix.

That was not what I intended. The building blocks specific to the main part of the draft would of course be specified in the main part of the draft. Recommendations specific to the appendix (such as how to handle GET/POST at a C-H-Proxy) would go to the appendix.